### PR TITLE
BUG: Update and enable ITKv3 Examples

### PR DESCRIPTION
**Pull request by blowekamp**
## PR Checklist
- [ ] [Makes breaking changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes)
- [ ] [Makes design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes)
- [ ] Adds the License notice to new files.
- [ ] Adds Python wrapping.
- [ ] Adds tests and baseline comparison (quantitative).
- [ ] [Adds test data](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Documentation/UploadBinaryData.md).
- [ ] Adds Examples to [ITKExamples](https://github.com/InsightSoftwareConsortium/ITKExamples)
- [ ] Adds Documentation.

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

File: Examples/RegistrationITKv3/CMakeLists.txt [hunk 1]
```diff
@@ -27,12 +27,16 @@ target_link_libraries(ITKv3ImageRegistration4 ${ITK_LIBRARIES})
 add_executable(ITKv3ImageRegistration7 ImageRegistration7.cxx )
 target_link_libraries(ITKv3ImageRegistration7 ${ITK_LIBRARIES})
 
+
 add_executable(ITKv3ImageRegistration11 ImageRegistration11.cxx )
 target_link_libraries(ITKv3ImageRegistration11 ${ITK_LIBRARIES})
 
+# This example in getting stuck and timing out, likely due to spatial
```
Comment (hjmjohnson) [inline Examples/RegistrationITKv3/CMakeLists.txt]: Incorporated into #813

File: Examples/CMakeLists.txt [hunk 2]
```diff
@@ -17,6 +17,4 @@ add_subdirectory(SpatialObjects)
 add_subdirectory(Statistics)
 add_subdirectory(RegistrationITKv4)
 
-if(ITKV3_COMPATIBILITY)
```
Comment (hjmjohnson) [inline Examples/CMakeLists.txt]: Moved to Modules/Registration/Common/test in #813

File: Examples/RegistrationITKv3/CMakeLists.txt [hunk 3]
```diff
@@ -79,13 +83,13 @@ target_link_libraries(ITKv3MultiResImageRegistration2 ${ITK_LIBRARIES})
 
 add_executable(ITKv3MultiResImageRegistration3 MultiResImageRegistration3.cxx )
 target_link_libraries(ITKv3MultiResImageRegistration3 ${ITK_LIBRARIES})
-
+if (ITKFEMRegistration_ENABLED)
```
Comment (hjmjohnson) [inline Examples/RegistrationITKv3/CMakeLists.txt]: This is not necessary.  These are  100% redundant with ITKv4 registration examples.

File: Examples/RegistrationITKv3/DeformationFieldJacobian.cxx [hunk 4]
```diff
@@ -19,7 +19,7 @@
 #include "itkImageFileReader.h"
 #include "itkImageFileWriter.h"
 
-#include "itkDeformationFieldJacobianDeterminantFilter.h"
```
Comment (hjmjohnson) [inline Examples/RegistrationITKv3/DeformationFieldJacobian.cxx]: Also done in #813.

File: Examples/RegistrationITKv3/ImageRegistrationHistogramPlotter.cxx [hunk 5]
```diff
@@ -203,7 +203,7 @@ class HistogramWriter
   using OutputPixelType = HistogramToEntropyImageFilterType::OutputPixelType;
 
   HistogramWriter():
-    m_Metric(0)
```
Comment (hjmjohnson) [inline Examples/RegistrationITKv3/ImageRegistrationHistogramPlotter.cxx]: Duplicate of ITKv4 example. Not needed here.

**Comment by blowekamp** [comment]
dup of #813.

**Pull request closed without merging**